BootloaderCorePkg/Library/SmbiosInitLib/SmbiosInitLib.c

```diff
@@ -16,46 +16,71 @@
 #include <Library/BootloaderCommonLib.h>
 #include "SmbiosTables.h"
 
-VOID      *mType127Ptr            =   NULL;
+VOID    *mType127Ptr      =   NULL;
+//
+// Add more platform specific strings
+// in the following format
+//
+SMBIOS_TYPE_STRINGS  DefaultSmbiosStrings[] = {
+  // Type                StrId   String
+  // Type 0
+  {  SMBIOS_TYPE_BIOS_INFORMATION ,    1,  "Intel Corporation"    },  // Vendor
+  {  SMBIOS_TYPE_BIOS_INFORMATION ,    2,  "SB_XXXX.XXX.XXX.XXX"  },  // BIOS Version
+  {  SMBIOS_TYPE_BIOS_INFORMATION ,    3,  "Unknown Date"       },  // BIOS Release Date
+  // Type 1
+  {  SMBIOS_TYPE_SYSTEM_INFORMATION ,    1,  "Intel Corporation"    },  // Manufacturer
+  {  SMBIOS_TYPE_SYSTEM_INFORMATION ,    2,  "Unknown Platform"     },  // Product Name
+  {  SMBIOS_TYPE_SYSTEM_INFORMATION ,    3,  "Platform ver-XXX.XXX.XXXXX.X"     },  // Version
+  {  SMBIOS_TYPE_SYSTEM_INFORMATION ,    4,  "Unknown Serial no",   },  // Serial Number
+  {  SMBIOS_TYPE_SYSTEM_INFORMATION ,    5,  "Unknown SKU"      },  // SKU Variant
+  {  SMBIOS_TYPE_SYSTEM_INFORMATION ,    6,  "Client Platform",     },  // Serial Number
+  // Type 2
+  {  SMBIOS_TYPE_BASEBOARD_INFORMATION ,   1,  "Unknown Serial no"   },  // Manufacturer
+  {  SMBIOS_TYPE_BASEBOARD_INFORMATION ,   2,  "Unknown Product"     },  // Product Name
+  {  SMBIOS_TYPE_BASEBOARD_INFORMATION ,   3,  "Build ver X-XXXXXXXXXXXXXXXXXX"    },  // Version
+  {  SMBIOS_TYPE_BASEBOARD_INFORMATION ,   4,  "Unknown ProdNumber",   },  // Serial Number
+  // Type 127 - End of strings
+  {   SMBIOS_TYPE_END_OF_TABLE,      0,  "End Of String Array"   },
+};
 
 /**
   Check if the Smbios types (including the entry point struct)
   have crossed the statically allocated size for Smbios init
 
   @param[in]  NewLength   New TypeLength to be checked for overflow if added
 
-  @retval                 EFI_DEVICE_ERROR    , if Smbios Entry is NULL
-                          EFI_BUFFER_TOO_SMALL, if new taype to be added exceeds allocated memory
-                          EFI_SUCCESS         , if new type can be added without mem overflow
+  @retval         EFI_DEVICE_ERROR  , if Smbios Entry is NULL
+              EFI_BUFFER_TOO_SMALL, if new taype to be added exceeds allocated memory
+              EFI_SUCCESS     , if new type can be added without mem overflow
 
 **/
 EFI_STATUS
 EFIAPI
 CheckSmbiosOverflow (
-  IN  UINT16    NewLength
+  IN  UINT16  NewLength
   )
 {
-  SMBIOS_TABLE_ENTRY_POINT      *SmbiosEntry;
+  SMBIOS_TABLE_ENTRY_POINT    *SmbiosEntry;

```

Comment:
This function did not change at all.
Please keep the original indent.

Reply:
I had fixed it. its due to my editor settings and uploaded new patches.